Add test info to README (#889)
aiidateam/aiida-quantumespresso: README.md

List the installed pseudopotential families with the command `aiida-pseudo list`.
You can then use the name of any family in the command line using the `-F` flag.

## Development

> ### Test

Reviewer: ```suggestion
### Running tests
```

> ```shell

Reviewer: Maybe preface this with "To run the tests, simply clone and install the package locally with the `[tests]` optional dependencies:"

git clone https://github.com/aiidateam/aiida-quantumespresso .
cd aiida-quantumespresso
pip install -e .[tests]  # install extra dependencies for test

> pytest tests/calculations/test_pw.py # run tests

Reviewer: Maybe we want to show how to run all tests by default?

```suggestion
pytest # run tests
```

You can also add an example to show how to run a specific set of tests, but we should assume a developer understands the basics of `pytest`.

```

> One can also use `tox` to run the test. `tox` allows setting the Python version for the test.

Reviewer: ```suggestion
You can also use `tox` to run the test set. Here you can also use the `-e` option to specify the Python version for the test run:
```
